### PR TITLE
Removes Commie Costume and PMC Corpse for thematic clothing and corpse on Soro

### PR DESCRIPTION
**Pull request by PalJohnson**
## About The Pull Request

Removes old ss13 soviet uniform sprite and replaces it with updated soviet sprite and appropriate security locker for soroskye. Removes old PMC spawn corpse with upp soldier corpse.

## Why It's Good For The Game

The map was made when it didn't have proper corpses or uniform sprites. This seeks to address the out-of-date content.

New security locker contains what you expect from it but instead of a station security uniform and security vest it has the soviet uniform and soviet winter coat with pistol belt.

## Changelog

:cl:
add: Added new soro security locker with contents to match the map
qol: removed old commie uniform and replaced it with updated sprite and locker
fix: fixed pmc corpse from spawning and replaced with upp corpse
/:cl:

### CODE REVIEW

**Review by Agameofscones — APPROVED** [review]
mapping checks out

**Review by stanalbatross — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -77,6 +77,38 @@
...
+	new /obj/item/reagent_container/spray/pepper(src)
+	new /obj/item/explosive/grenade/flashbang(src)
+	new /obj/item/weapon/melee/baton/loaded(src)
+	new /obj/item/weapon/gun/energy/taser(src)
+	new /obj/item/clothing/glasses/sunglasses/sechud(src)
+	new /obj/item/device/hailer(src)
+	new /obj/item/clothing/accessory/storage/black_vest(src)
+	new /obj/item/clothing/mask/rebreather/scarf(src)
+	new /obj/item/clothing/under/rank/veteran/soviet_uniform_01(src)
+	new /obj/item/storage/belt/gun/type47/NY(src)
```
[the remark this answers is not in the document]
Comment (PalJohnson) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: created new locker so it doesn't inherent old code and loaodut

image

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -77,6 +77,38 @@
 	new /obj/item/clothing/head/soft/sec/corp(src)
 	new /obj/item/clothing/under/rank/security/corp(src)
 
+/obj/structure/closet/secure_closet/soro/security
```
Comment (stanalbatross) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:80]: don't make it a child of a non existent parent, should be like /soro_security or simply use inheritance properly and move /security into /security/standard to keep icon shit the same

**Review by stanalbatross — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 2 above)
Comment (PalJohnson) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: applied changes

**Review by ThePiachu — APPROVED** [review]

**Pull request merged**